Executable/Program.cs

```diff
@@ -46,67 +54,36 @@ internal static void Main(string[] args)
                     Environment.Exit(1);
                 }
 
-                string windowsDirectory = Path.Combine(Environment.CurrentDirectory, "../..");
-                string macDirectory = Path.Combine(Environment.CurrentDirectory, "../../../../..");
+                GetInfo(out os, out string windowsDirectory, out string macDirectory);
```

Comment:
Why have the directory variable be separate for each OS? Wouldn't it be fine if it was just a single variable?

Reply:
That is true, and it will also cut down on some if elses

Reply:
Addressed in commit 779c377